documentation.lisp

```diff
@@ -472,6 +472,86 @@ See ACTIVITY")
 
 See ACTIVITY")
 
+  (type announcement-account
+    "List of accounts related to an announcement
+
+See ANNOUNCEMENT
+See ID
+See USERNAME
+See ACCOUNT-NAME
+See URL")
+
+  (type announcement-status
+    "List of statuses related to an announcement
+
+See ANNOUNCEMENT
+See ID
+See URL")
+
+  (type announcement
+    "Representation of an administrator's announce
+
+See ID
+See CONTENT
+See STARTS-AT
+See ENDS-AT
+See PUBLISHED
+See ALL-DAY
+See PUBLISHED-AT
+See UPDATED-AT
+See READP
+See MENTIONS
+See STATUSES
+See TAGS
+See EMOJIS
+See REACTIONS")
+
+  (function starts-at
+    "Returns a date when announce will start.")
```

Comment:
Please backreference the types that are relevant to the function as for other function docstrings.

Reply:
Hi!

Sorry, I missed that, I hope i have fixed that with the last commit.

Bye!
C.